libs/core.lua

```diff
@@ -129,6 +129,9 @@ local function makeCore(config)
       error("Not a package: " .. path)
     end
     local author, name, version = pkg.normalize(meta)
+    if not author or not name then
```

Comment:
Or even if it does optimize this, changing the logic seems easier to me to understand the intent.  What do you think?